packages/react/src/lib/positioner/Popper.tsx

```diff
@@ -127,7 +127,15 @@ const Popper: React.FunctionComponent<PopperProps> = props => {
       popperRef.current = createPopper(targetRef.current, contentRef.current, options)
     },
     // TODO review dependencies for popperHasScrollableParent
```

Comment:
is this comment still necessary - I mean, if these are reviewed, we may remove `TODO` comment :) or there are some checks that you still expect to be made?

Reply:
Not sure, this was here before. I found this issue while working on the tooltip, so I added the triggerRef there as well. I think is @Bugaa92 's call about whether it is safe to remove it